src/extensions/agent_modules/cloud_communication/eg_mqtt_broker_client/src/agentinfo_request_operation.c

```diff
@@ -195,7 +219,8 @@ static bool HandlingRequestAgentInfo(ADUC_Retriable_Operation_Context* context,
         ADUC_Retriable_Set_State(context, ADUC_Retriable_Operation_State_InProgress);
 
         // is the current request timed-out?
-        if (context->lastExecutionTime + context->operationTimeoutSecs < nowTime)
+        // TODO: use context->operationTimeoutSecs instead of 30 once operationTimeoutSecs set correctly to non-zero.
```

Comment:
> operationTimeoutSecs set correctly to non-zero [](http://example.com/codeflow?start=69&length=47)

server side?

Reply:
It is currently a default setting in the client side, but we are having conversations about also having poll freq and retry/timeout settings flow down from service.